R/fcn_computeQC.R

```diff
@@ -79,7 +79,7 @@ report_version = paste0("v", pv)
 
 time_start = Sys.time()
 
-stats_file = paste(txt_folder, "\\report_", report_version, "_stats.txt", sep="")
+stats_file = paste(txt_folder, paste("report_", report_version, "_stats.txt", sep=""), sep=.Platform$file.sep)
```

Comment:
I'd be in favour of doing it explicitly to ease reading:
paste0(txt_folder, .Platform$file.sep, "report_", report_version, "_stats.txt")